src/components/UiText/index.vue

```diff
@@ -64,7 +75,17 @@ export default {
      * and rounding value
      */
     step: { type: [Number, String], default: '' },
+    /**
+     * Show or hide icon for toggle view state. 
+     */
+    hideSwitch: { type: Boolean, default: false }
```

Comment:
What switch do you mean?
bad variable name. Rename pls

Reply:
I mean hide or show icon for toggling text show.
Name: hideShowBtn , is a better one?